index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>Project</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <header class ="header"><h3>Hello Microverse!</h3></header>
```

Comment:
- [ ] Make sure to use the h1 tag for your header. This was listed in the project requirements.
 Also, you should only use the h1 tag in the title of your website for semantics.
> You can read about this tip and more at this [link](https://www.w3schools.com/html/html5_semantic_elements.asp)